index.bs

```diff
@@ -4252,6 +4268,10 @@ The following abstract operations operate on {{WritableStream}} instances at a h
  id="writable-stream-abort">WritableStreamAbort(|stream|, |reason|)</dfn> performs the following
  steps:
 
+ 1. If |stream|.[=WritableStream/[[state]]=] is "`closed`" or "`errored`", return
+    [=a promise resolved with=] undefined.
+ 1. [=Signal abort=] on
```

Comment:
I guess technically it could throw if some spec added a throwing algorithm to the "abort algorithms" list, but that seems unlikely. I filed https://github.com/whatwg/dom/issues/984 on making it impossible.